Formula/msgpack.rb

```diff
@@ -63,7 +64,7 @@ def install
       }
     EOS
 
-    system ENV.cc, "-o", "test", "test.c", "-L#{lib}", "-lmsgpackc"
+    system ENV.cc, "-o", "test", "test.c", "-L#{lib}", "-lmsgpack-c"
```

Comment:
If upstream renamed the library then that will break all the dependents. We should check with upstream whether this is intentional.

Reply:
This was done per https://github.com/msgpack/msgpack-c/pull/1053 to address [the issue](https://github.com/msgpack/msgpack-c/issues/1043)

Reply:
In that case this needs to be reported to all dependent projects since it won't be enough to bump their revisions.

Reply:
yeah, I think your symlink alias change makes more sense.

Reply:
because of the linkage test failures, the revision bumps were mostly just enforcing the rebuilds.